src/main/java/com/ensao/gi5/lint/rules/BooleanExpressionsRule.java

```diff
@@ -0,0 +1,35 @@
+package com.ensao.gi5.lint.rules;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.rules.violations.Violation;
+import com.ensao.gi5.lint.rules.violations.ViolationFactory;
+import com.ensao.gi5.lint.visitor.BooleanExpressionsVisitor;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+import com.github.javaparser.ast.expr.BinaryExpr;
+
+public class BooleanExpressionsRule extends Rule {
+
+	public BooleanExpressionsRule() {
+		super(Constantes.LINT_REG_006, Level.HIGHEST);
+	}
+
+	@Override
+	public void apply(CompilationUnitWrapper compilationUnit) {
+		Set<BinaryExpr> bExps = new LinkedHashSet<BinaryExpr>();
```

Comment:
could be `Set<BinaryExpr> bExps = new LinkedHashSet<>();`

Reply:
I appreciate your relevant remarks sir and i promess i will take them into consideration.